### PR TITLE
Add Getting Started File

### PR DESCRIPTION
**Pull request by git-create-devben**
I add a file name GettingStarting.md where I simplify how to contribute with details step and add some image to show how to contribute and I remove how to contribute from README.md to GettingStarted.md, In the readme file on how to contribute I link it to GettingStarting files.

Am open for any corrections.

fix issue #91

### CODE REVIEW

File: Gettingstarted.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+![Point to a star button](/assets/readme/StarRepo.png)
+
+2.**Fork this repository** (You can do this by clicking the **Fork** button on this repository's page)
+
+![Point to a fork button](/assets/readme/forkRepo.png)
+
+**Clone this repository** to your local manchine. Open a termianl navigate to the directory where you want to clone this repository. Then, run the following command:
+
+`https://github.com/<Your_username>/WWCodeGreenville.git`
```
Comment (klmansel) [inline Gettingstarted.md:15]:
```suggestion
`https://github.com/<your_username>/WWCodeGreenville.git`
```

File: Gettingstarted.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+1.**Star  this repository** (Click the **Star** button at the top of this page)
+
+![Point to a star button](/assets/readme/StarRepo.png)
+
+2.**Fork this repository** (You can do this by clicking the **Fork** button on this repository's page)
+
+![Point to a fork button](/assets/readme/forkRepo.png)
+
+**Clone this repository** to your local manchine. Open a termianl navigate to the directory where you want to clone this repository. Then, run the following command:
```
Comment (klmansel) [inline Gettingstarted.md:13]:
```suggestion
**Clone this repository** to your local machine. Open a terminal and navigate to the directory where you want to clone this repository. Then, run the following command:
```

File: Gettingstarted.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+`https://github.com/<Your_username>/WWCodeGreenville.git`
+
+Replace <your_username> with your GitHub username
```
Comment (klmansel) [inline Gettingstarted.md:17]:
```suggestion
Replace `<your_username>` with your GitHub username
```

File: Gettingstarted.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+3.**Create a new branch**. To do this, run the following command:
+
+`git checkout -b <branch_name>`
+
+Replace <branch_name> with the name of the branch that you want to create.
```
Comment (klmansel) [inline Gettingstarted.md:23]:
```suggestion
Replace `<branch_name>` with the name of the branch that you want to create.
```

File: Gettingstarted.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+4.**Make your changes**. Open the file(s) that you want to edit in any code editor and make your changes.
+
+5.**Commit your changes**. To do this, run the following command:
+
+`git commit -m "<commit_message>"`
+
+Replace <commit_message> with a short message that describes your changes.
```
Comment (klmansel) [inline Gettingstarted.md:31]:
```suggestion
Replace `<commit_message>` with a short message that describes your changes.
```

File: Gettingstarted.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+6.**Push your changes to your forked repository**. To do this, run the following command:
+
+`git push origin <branch_name>`
+
+Replace <branch_name> with the name of the branch that you want to pushed your changes to.
```
Comment (klmansel) [inline Gettingstarted.md:37]:
```suggestion
Replace `<branch_name>` with the name of the branch that you want to pushed your changes to.
```

File: Gettingstarted.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+
+7.**Create a pull request**. Go to the repository page that you forked and click the **Compare and pull request**.
```
Comment (klmansel) [inline Gettingstarted.md:39]:
```suggestion
7.**Create a pull request**. Go to the repository page that you forked and click the **Compare and pull request** button
```

File: Gettingstarted.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+![Point to a Pull request button](/assets/readme/compare-and-pull-request.png)
+
+8.**Respond to feedback**. we may review your changes and suggest improvements. so try to Respond to our feedback and make any necessary changes.
```
Comment (klmansel) [inline Gettingstarted.md:43]:
```suggestion
8.**Respond to feedback**. We may review your changes and suggest improvements, so try to respond to our feedback and make any necessary changes.
```

File: Gettingstarted.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
...
+
+9.**Merge your pull request**. Once we're happy with your changes, they we will merge your pull request into the main branch of the repository.
```
Comment (klmansel) [inline Gettingstarted.md:45]:
```suggestion
9.**Merge your pull request**. Once we're happy with your changes, we will merge your pull request into the main branch of the repository.
```

File: README.md [hunk 10]
```diff
@@ -38,13 +38,10 @@ If there are any questions, please let us know 😁
...
 <span id="hacktoberfest">![Hacktoberfest][hacktoberfest-heading]</span>
 
-- 🍴 To get started, [fork this repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
-- 👯‍♀️ [Clone the repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your computer.
-- 👩🏽‍💻 Participate by requesting to be assigned to our open issues and contributing to our docs.
-- 🤓 You must have 4 PRs (pull requests) merged to complete the challenge!
-- 🌲 You will get a tree planted in your name and some cool virtual swag!
-- 🎃 Join our network's Slack channel #hacktober-2023 and ping one of our Leads for help or just stop by and chat!
-- 👀 Check the [WWCode Greenville Notion Doc](https://wwcode.notion.site/Hacktoberfest-2023-d690520f41a04f1e956abf475c2313db?pvs=4) if you need help with GitHub or contributing to this project.
+## Getting started
```
Comment (klmansel) [inline README.md:41]: @git-create-devben Let's leave the Hacktoberfest section in and add the Getting Started section, but with a heading that matches the others and a link from the Table of Contents. We'll remove the Hacktoberfest section after this month.

**Review by klmansel — CHANGES_REQUESTED** [review]
A few small changes in the Readme, and I love the creativity of your profile! 
Can you move that to a separate PR that closes #68 ? Thanks for your contributions!

File: README.md (hunk 10 above)
Comment (git-create-devben) [reply README.md:41]: Thanks for your suggestions, can you please clarify what you by "link from the Table of Contents"

Comment (git-create-devben) [reply README.md:41]: Thanks for the suggestions, I can see the installation and prerequisites are sub list under the hacktoberfest2023 so am adding Getting Started section after prerequisite. is that okey or I should just leave it the way it's above the installation.

File: README.md [hunk 11]
```diff
@@ -63,6 +65,11 @@ There are two ways to use git to interact with our repository: command-line Git
 1. Git Command Line: There are many free tutorials that can walk you through the commands.  [FreeCodeCamp](https://www.freecodecamp.org/news/git-and-github-for-beginners/) has a great beginner tutorial.
 2. Desktop Git App: Another way that users can interact with our repository is through a desktop app like Github's own [Desktop App](https://desktop.github.com/), which has simple UI and can be intuitive for first time users.
 
+<!-- Getting Started -->
+## Getting started
+<span id="gettingstarted">![GettingStarted][getting-started-heading]</span>
+To learn how to contribute to this project, please see the [getting started guide](./GettingStarted.md).
```
Comment (klmansel) [inline README.md:71]: @git-create-devben can you add a newline after the heading here so the description can be on the next line?

**Review by klmansel — CHANGES_REQUESTED** [review]
One small change and I'll get this approved and merged - thanks!

**Comment by JenBauer** [comment]
I think the fix in the other PR was meant to go with this one?

**Comment by git-create-devben** [comment]
She didn't merge this one so she merge the second branch and I push the changes to the second branch. is it showing some kind of error or conflict ?

**Comment by klmansel** [comment]
@git-create-devben sorry about that! can you reopen the 404 PR and I'll accept that one! Thank you ⭐️

**Pull request closed without merging**